catalog/env.template

```diff
@@ -75,7 +75,7 @@ EMR_TEST_CONN_ID=emr_test
 
 # Connection to the Ingestion Server, used for managing data refreshes. Default is used to
 # connect to your locally running ingestion server.
-AIRFLOW_CONN_DATA_REFRESH=http://172.17.0.1:8001
+AIRFLOW_CONN_DATA_REFRESH=http://ingestion_server:8001
```

Comment:
This change is now possible thanks to the monorepo effort and no longer requires different settings for different OS environments. We can just go straight through the docker network now.